media-libs/libbluray/libbluray-1.3.1.ebuild

```diff
@@ -31,7 +31,7 @@ RDEPEND="
 "
 DEPEND="
 	${RDEPEND}
-	java? ( >=virtual/jdk-1.8:* )
+	java? ( >=virtual/jre-1.8:* )
```

Comment:
in DEPEND I'd keep jdk but remove java stuff from BDEPEND ( unless it's completely different for `java-pkg-opt-2` ).

Reply:
Ahh right, I confused DEPEND and RDEPEND. I think since this uses JNA, having JDK in BDEPEND and DEPEND is probably fine.